### DISCUSSION TITLE
Idea: LinkedIn shield

### DISCUSSION THREAD

**Discussion by adamlui** [badges/shields#10012] [Ideas]
LinkedIn is quite popular w/ over 900 million users, a follower shield would be sweet and likely used alot

**Comment by jNullj** [comment]
It looks like LinkedIn has an API, didn't go into how to get access it yet.
Can you better describe what the badge should look like, you can use our [static badge](https://shields.io/badges) to create an example badge.
Also what type of followers do you intend the badge to show? User followers or page followers?

**Reply by adamlui** [reply]
Yes it should look like this, to show number of users that follow you (just like other social platform shields): 

 ![](https://img.shields.io/badge/Follow%20in%2fadamlui-1.1k-blue?logo=linkedin&style=social "Follow in/adamlui on LinkedIn")

Source:
`![](https://img.shields.io/badge/Follow%20in%2fadamlui-1.1k-blue?logo=linkedin&style=social "Follow in/adamlui on LinkedIn")`

**Reply by adamlui** [reply]
I would use if it showed a Company Page's follower count, too (to promote Company Page)

**Comment by jNullj** [comment]
After looking into the API it looks like there is no access to users followers.
Regarding company followers the shields instance has to have API access with ADMINISTRATOR permissions to the company page.

Those make it not practical to implement the badge unless upstream adds support for those calls.
As described in the contribution guide we try to keep away from web scrapping for stability. Also it appears that without a user the follower counter is not very accurate.

The bottom line is, its unlikely we add those badges until upstream API make the data accessible.
If in the future you see changes or find another source to obtain the data please notify us and I think someone might be glad to add these.

**Reply by adamlui** [reply]
Ok how come this is a Closed idea though? Ideas are just that, open to any form of solution

Also I didn't get what this means "without a user the follower counter is not very accurate"

**Reply by jNullj** [reply]
Wanted to show in a screenshot how the followers is a rounded value, but suddenly it appears to be exact, not sure why it changed but my bad for getting too quickly into conclusion about followers accuracy. might be related somehow to using another browser/os.
Also i re-opened the idea, thought about it like an open issues, i rarely use the discussions option.
I hope someone might have a solution.

**Comment by chris48s** [comment]
Refs https://github.com/badges/shields/issues/7177
